src/devtools/views/Components/HooksTree.js

```diff
@@ -71,6 +72,11 @@ type HookViewProps = {|
 
 function HookView({ canEditHooks, hook, id, path = [] }: HookViewProps) {
   const { name, id: hookID, isStateEditable, subHooks, value } = hook;
+  if (hook.hasOwnProperty(meta.inspected)) {
+    // This Hook is too deep and hasn't been hydrated.
+    // TODO: show UI to load its data.
+    return null;
+  }
```

Comment:
This early return needs to happen after `useContext` below.

I think we should also render _something_ to indicate that we've truncated (even if just "...").

I also think we'll need to handle this general case for deep props/state/context.

(I'll address the lint thing as part of adding a temporary truncation indicator.)

![Screen Shot 2019-04-04 at 2 08 25 PM](https://user-images.githubusercontent.com/29597/55588642-22079d00-56e3-11e9-9caf-3306173265e8.png)